.github/workflows/test-down-level.yml

```
@@ -23,7 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        release: ['2.0.2', '2.1.1']
+        release: ['2.0.2', '2.1.1', '2.1.2']
```

```suggestion
        release: ['2.0.2', '2.1.2']
```
Replace previous patch version. We only need to test the latest for each official release.